common/lib/transport/connectionmanager.js

```diff
@@ -551,6 +551,11 @@ var ConnectionManager = (function() {
 				state = 'disconnected';
 			}
 			this.notifyState({state: state, error: error});
+		} else if(wasActive) {
```

Comment:
Personally I'd prefer a single `if(wasActive)` with both of these cases as sub-cases